### PR TITLE
Addition of new Pachyderm tests

### PR DESCRIPTION
**Pull request by jonandernovella**
This PR includes new tests to verify if Pachyderm was successfully deployed.

### CODE REVIEW

**Comment by jonandernovella** [comment]
@andersla LGTM. Needed to change the ddefault passwords of the playbooks tho.

**Comment by andersla** [comment]
Hi @jonandernovella I was thinking a test that only runs when travis is running, but I guess this is good too:-)

File: playbooks/roles/pachyderm/tasks/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
+     --set pachd.image.tag="{{ pachyderm_tag }}",pachd.worker.tag="{{ pachyderm_tag }}",etcd.image.tag="{{ etcd_tag }}",credentials=s3,s3.accessKey="{{ pachyderm_minio_accesskey }}",s3.secretKey="{{ pachyderm_minio_secretkey }}",s3.bucketName=defaultbucket,s3.endpoint="{{ minio_release_name }}"-minio-svc.default.svc.cluster.local:9000,s3.signature=0,etcd.persistence.enabled=true,persistence.storageClass="{{ os_sc }}",etcd.persistence.accessMode=ReadWriteMany,etcd.persistence.size="{{ pachyderm_etcd_pvc_size }}",etcd.resources.requests.cpu=1,etcd.resources.requests.memory=2G,pachd.resources.requests.cpu=1,pachd.resources.requests.memory=2G,pachd.pfsCache=1G
      "{{ pachyderm_release_name }}" stable/pachyderm
 
  - name: wait for etcd to be Running
```
Comment (andersla) [inline playbooks/roles/pachyderm/tasks/main.yml]: Why is it called wait for etcd running?

Comment (jonandernovella) [reply playbooks/roles/pachyderm/tasks/main.yml]: @andersla because `etcd` needs to be running before `pachd`. It's part of the deployment.

Comment (andersla) [reply playbooks/roles/pachyderm/tasks/main.yml]: I just didn't look at the whole code...;)

**Comment by andersla** [comment]
I changed the uncommenting part in .travis

**Comment by andersla** [comment]
Jättebra @jonandernovella !

**Pull request merged**